src/app/pages/admin-panel/admin-panel.component.html

```diff
@@ -1,3 +1,4 @@
+<!-- TODO: header and sidebar should be added to a single, top-level page of the application, usually app.component.html -->
```

Comment:
It's quite unusual and does not look right to me that you embed header and sidebar into every separate page
Usually header and sidebar are located in a top-level component, on the same level with a router outlet

If you need to have some pages without header and sidebar (like login for example) - then it's done by having two top-level components/containers, or by conditionally hiding/showing header/sidebar based on current route